operations/patch_util_test.go

```diff
@@ -0,0 +1,144 @@
+package operations
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+)
+
+const testConfigFile = ".evergreen.yml"
```

Comment:
I'd just put this as a parameter that's set in setup suite.

Reply:
Done.